config/routes.rb

```diff
@@ -5,6 +5,11 @@
   post '/login', to: "sessions#create"
   get '/breeds', to: "breeds#index"
   get '/breeds/:name', to: "breeds#show", as: "breed"
-  get '/cats/:id', to: "cats#show"
+  get '/cats/:id', to: "cats#show", as: "cat"
 
+  scope '/admin' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.